be/src/column/column_access_path.cpp

```diff
@@ -66,22 +69,13 @@ Status ColumnAccessPath::init(const std::string& parent_path, const TColumnAcces
 
     for (const auto& child : column_path.children) {
         ColumnAccessPathPtr child_path = std::make_unique<ColumnAccessPath>();
-        RETURN_IF_ERROR(child_path->init(_absolute_path + "/", child, state, pool));
+        RETURN_IF_ERROR(child_path->init(_absolute_path + ".", child, state, pool));
```

Comment:
Is if work for such case?

```
'{".key": "value"}'
```

Reply:
This is generated by fe and will add a '.' to the path, will be`\".key\"`